docs/containers-storage-applydiff-using-staging-dir.md

```diff
@@ -0,0 +1,27 @@
+## containers-storage-applydiff-using-staging-dir 1 "September 2023"
```

Comment:
There should be a reference to this in the SUB-COMMANDS section of "docs/containers-storage.md".

Reply:
thanks, added now